src/lib/Poster.svelte

```diff
@@ -20,7 +20,9 @@
     name?: string;
     overview?: string;
     id: number; // tmdb id
-    media_type: MediaType;
+    media_type: MediaType; 
+    release_date?: string;
```

Comment:
This is good, though it doesn't work on the main watched list page (`src/routes/(app)/+page.svelte`).

The Poster media prop is passed manually in there, so something like this needs to be added there (in the media object):
```
release_date: w.content.release_date
```

I think you may also need to update the Content type in types.ts to include release_date.

Reply:
Thanks for all the feedback. I'll implement the changes asap.

Reply:
Done!